ml-agents/mlagents/trainers/components/bc/module.py

```diff
@@ -22,7 +22,7 @@ def __init__(
         samples_per_update: int = 0,
     ):
         """
-        A BC trainer that can be used inline with RL, especially for pretraining.
+        A BC trainer that can be used inline with RL, especially for behavioral_cloning.
```

Comment:
Was this a find-and-replace? sounds awkward here.

Reply:
Removed this, as it no longer makes sense